app/vader_sentiment.py

```diff
@@ -1,20 +1,14 @@
 from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
-vader = SentimentIntensityAnalyzer()
-
+analyzer = SentimentIntensityAnalyzer()
 
 def vader_score(text: str):
     """
```

Comment:
This change breaks the existing endpoint we have for doing sentiment analysis. If you need different behavior it would be better to write an addition function rather than drastically changing an existing one.

Reply:
I went ahead and reverted it back to the original. In my notebook, I have a way of getting the actual scores for the visualizations.